cms/djangoapps/contentstore/tests/test_courseware_index.py

```diff
@@ -29,6 +33,12 @@
 from contentstore.courseware_index import CoursewareSearchIndexer, LibrarySearchIndexer, SearchIndexingError
 from contentstore.signals import listen_for_course_publish, listen_for_library_update
 
+import json
```

Comment:
I'd prefer to see the order of the imports in sensible grouping:
see https://github.com/edx/edx-platform/wiki/Python-Guidelines#imports-order